botocore/endpoint_provider.py

```diff
@@ -577,6 +577,7 @@ class ParameterType(Enum):
 
     string = str
     boolean = bool
+    stringarray = tuple
```

Comment:
Why not use `string_array` to be more idiomatic?

Reply:
This comes straight from the model and needs to match what is in there.  We can do a transformation on it if that makes sense; right now we are only calling `lower()` on it.  Making it `string_array` would require us to do do a more complex transformation, which I prefer to keep simple unless we have a good reason to make it more idiomatic.

I'm not closed off to the idea though

Reply:
Got it, I think it makes sense to leave it as `stringarray` then. Thanks